packages/mui-material/src/Chip/Chip.test.js

```diff
@@ -733,4 +733,36 @@ describe('<Chip />', () => {
       ).not.to.throw();
     });
   });
+
+  it('should not throw on clicking Chip when onClick is not provided', () => {
+    expect(() => {
+      const { getByTestId } = render(<Chip data-testid="chip" />);
+      const chip = getByTestId('chip');
+      fireEvent.click(chip);
+    }).not.throw();
+  });
+
+  it('should not throw on keydown when onKeyDown is not provided', () => {
+    expect(() => {
+      const { getByTestId } = render(<Chip data-testid="chip" onClick={() => {}} />);
+      const chip = getByTestId('chip');
+      act(() => {
+        chip.focus();
+      });
+
+      fireEvent.keyDown(chip, { key: 'Enter' });
```

Comment:
As `keyDown` can't be fired on non-focusable elements, i converted `Chip` to `button` by adding `onClick={()=>{}}` and then fired `keyDown` event